### ISSUE TITLE
Reduce dependencies

### ISSUE THREAD

**Issue by Enngage** [webcat12345/ngx-intl-tel-input#163]
I just came across this library and while it looks awesome, having Bootstrap as a dependency is greatly discouraging. If you were able to provide dependency-less package, this would be really awesome and would gain so much more traction.

**Comment by pasevin** [comment]
Yes, I agree. This was my initial doubt about this library. I always look for clean and flexible implementations, which don't impose design decisions on me.

I'm currently swamped with my own project, so don't think I will be able to do this, but maybe someone else would take this on :)

**Comment by TSkills** [comment]
I agree, deps made me pass my way.
Nice work though

**Comment by elyas-bhy** [comment]
Same thoughts as @pasevin. Is this issue planned for anytime soon?

**Comment by all2pie** [comment]
Yep especially remove the Bootstrap and you can switch from `google-libphonenumber` to `libphonenumber-js`  for validation and make that internal dependency
